### DISCUSSION TITLE
Meetup.com zum organisieren der meetups

### DISCUSSION THREAD

**Discussion by thomaskvnze** [scheffle/hamburg_audio_dev_stammtisch#3] [General]
Hey! 

Ich habe jetzt mal geschaut, ob meetup.com kostenlos ist.
Leider gibt es tatsächlich keine kostenlose Variante. Es kostet 11,66$/Monat pro Monat wenn man es für min. 6 Monate bucht.
Ich würde das mal für ein halbes Jahr sponsorn und gucken, ob es mit der Organisation und Reichweite hilft. Ich wollte das sowieso mal ausprobieren. Es soll dabei helfen:

* Anmeldeliste führen, um zu wissen, wieviele Leute kommen
* Event Ankündigung und Änderungen über mail und push notifications
* Leichter öffentlich auffindbar, da ich meetup.com als "die" global bekannte Plattform zum auffinden von Meetups kenne.

Was haltet ihr davon?

**Comment by georgco** [comment]
Hi Thomas, vielen Dank für dein Sponsoring! 🙂👍 Wir können das ja gerne mal ausprobieren.

**Reply by thomaskvnze** [reply]
Ich werds mal am Wochenende aufsetzen! 💯

**Reply by georgco** [reply]
👍

**Reply by thomaskvnze** [reply]
Ich habe jetzt die Gruppe erstellt. Sobald sie genehmigt ist, kann ich den Link teilen.
Ich müsste dann noch wissen, wen ich als Organisator einladen kann.

Wann sollte noch einmal das nächste Event stattfinden?

**Reply by okriwat-fender** [reply]
Auf der Startseite steht ja "26.09.2024 19 Uhr"

**Reply by georgco** [reply]
> Ich habe jetzt die Gruppe erstellt. Sobald sie genehmigt ist, kann ich den Link teilen. Ich müsste dann noch wissen, wen ich als Organisator einladen kann.
> 
> Wann sollte noch einmal das nächste Event stattfinden?

Hey Thomas, super - vielen Dank! Ich habe mich jetzt auch auf meetup als "georgco" angemeldet. Du kannst mich da gerne als Mit-Organisator eintragen. Das nächste Treffen findet wie gesagt am 26.09.24 um 19 Uhr statt.

**Reply by thomaskvnze** [reply]
Danke! Ich hab das Event in der Meetup Gruppe erstellt. Es gibt auch eine Diskussionsfunktion unten im Event und Gruppenzugehörige können per email über relevante Infos benachrichtigt werden. 

Hier ist das Event:
https://www.meetup.com/de-DE/hamburg-audio-developer/events/303081807/?slug=hamburg-audio-developer&isFirstPublish=true

Hier ist die Gruppe:
https://www.meetup.com/de-DE/hamburg-audio-developer/?eventOrigin=event_home_page

Die Links können ja ins README.md gesetzt werden.

**Reply by georgco** [reply]
Moin! Ich weiß nicht, ob ihr jetzt schon alle auf meetup seid. Am 26.09.24 findet ja unser nächstes Treffen statt. Sollen wir da mal das "tanzende Einhorn" ausprobieren? Muss man da reservieren? Oder habt ihr andere Vorschläge?

**Reply by okriwat-fender** [reply]
Oh nein, am 26.09. ist im Einhorn "PubQuiz Abend mit QuizInTheNorth", also quizzen mit 6er Teams und Preisen und so. An solchen Tagen **kann man keine Tische reservieren** und es könnte auch recht voll werden.

Ich würde morgen noch mal die Werbetrommel rühren, da wäre ein Ort hilfreich :-)

**Reply by okriwat-fender** [reply]
Ich bin noch nicht dazu gekommen, Meetup einzurichten, aber was ist eigentlich aus unserem Januar Termin geworden 😆 ? Nicht einschlafen *alle rüttel*

**Reply by georgco** [reply]
Keine Sorge, ist nicht eingeschlafen. ;-) Ich hatte an Ende Februar gedacht für das nächste Treffen, z.B. den 27.02.25. Wir könnten ja diesmal das Alte Mädchen ausprobieren, was denkt ihr?

**Reply by okriwat-fender** [reply]
Von mir aus gerne. Kann das jemand bei Meetup vorschlagen bzw einrichten?

**Comment by georgco** [comment]
Hat keiner eine Idee? Es gab auch mal den Vorschlag, Kegeln zu gehen... :-)))

**Comment by thomaskvnze** [comment]
Hier sind 2 Vorschläge:

Altes Mädchen in der Sternschanze
Große Location, viele verschiedene Biere und es kann auch gegessen werden. https://altes-maedchen.com/en/

Ratsherrnbar
Liegt an einem Fluss, eher kleiner aber gemütlich. Wenn man eine größere Gruppe ist, kann man nach einem kleinen Bereich reservieren. Ich frage mal nach, ob das extra kostet.
https://ratsherrn.de/ratsherrn-gastro/#ratsherrnbar

**Comment by georgco** [comment]
Altes Mädchen klingt doch gut und unkompliziert. Lasst uns das machen! Dann müssen wir uns nur noch überlegen, wie uns die Leute erkennen, die uns noch nicht kennen (von Meetup). Eine rote Pappnase aufsetzen oder so... ;-)

**Reply by thomaskvnze** [reply]
Okay, ich ruf da heute mal an und kläre mal ab, dass wir dort genug Platz haben. Aktuell sind 6 Personen auf meetup angemeldet. Wisst ihr noch von Leuten, die kommen wollen und sich dort nicht angemeldet haben? Eine ungefähre Hausnummer wäre gut.

**Reply by okriwat-fender** [reply]
Lange Haare als Erkennungszeichen B) Super, dann trage ich das alte Mädchen schon mal weiter, also als Information

**Reply by okriwat-fender** [reply]
> Wisst ihr noch von Leuten, die kommen wollen und sich dort nicht angemeldet haben? Eine ungefähre Hausnummer wäre gut.

Ich frag bis 16 Uhr mal bei uns rum, da hat noch keiner MeetUp

**Reply by georgco** [reply]
Schwierig zu sagen, wie viele Leute kommen. Viele entscheiden sich spontan und auf meetup sind glaube ich auch noch nicht so viele. Ausgehend von den letzten beiden Treffen würde ich grob mit 10-15 Leuten rechnen.

**Reply by thomaskvnze** [reply]
Ich habe gerade mal eine Anfrage an das alte Mädchen bzgl. ihres "Stammtisch" Raumes gestellt. Der hat 23 Plätze und ist sogar mit Beamer etc. ausgestattet. Falls jemand irgendwann mal Lust hat einen interessanten Talk zu halten :-). Vielleicht kostet der nichts extra: https://altes-maedchen.com/en/stammtisch/

**Reply by okriwat-fender** [reply]
Von uns haben sich etwa 5 Leute angekündigt, leider ohne Talk :-)

**Reply by thomaskvnze** [reply]
Ich habe gerade eine Antwort vom alten Mädchen erhalten. Nächste Woche ist die WindEnergy Messe und sie nehmen keine Reservierungen mehr an. Ich gehe davon aus, dass das alte Mädchen kein Platz haben wird. Wir brauchen eine neue Location.

**Reply by georgco** [reply]
Das ist schade. Vielleicht kannst du nochmal bei der Ratsherrnbar anfragen, sofern es nichts extra kostet? Ansonsten ist es ja schwierig zu reservieren, wenn wir nicht verbindlich die Anzahl der Leute kennen. 

Zur Not würde ich dann wieder das BrewDog auf der Reeperbahn vorschlagen. Ist zwar auch nicht ideal, aber es gibt da eine lange Theke mit mehreren Stehhöckern.

**Reply by thomaskvnze** [reply]
Ich hab gestern noch eine Liste gemacht, die ich heute mal abklappere.

1. Ratsherrnbar
2. Sofabar
3. Kixx - kostet 4 Euro pro person, dafür sind dort ziemlich viele Tischkicker und eine Bar.
4. Hofbräu
5. BrewDog

**Reply by thomaskvnze** [reply]
Es ist leider alles reserviert und voll. Ich habe jetzt im Brewdog einfach 15 Plätze reserviert. Das wird schon klappen.

**Reply by thomaskvnze** [reply]
Ich schaue mal, dass ich den Stammtischraum im Alten Mädchen für das nächste Mal vorbuchen kann.

**Reply by georgco** [reply]
Alles klar, vielen Dank für deinen Einsatz!

**Reply by thomaskvnze** [reply]
@georgco Wollen wir den nächsten Termin im Januar ansetzen?

**Reply by georgco** [reply]
Moin Thomas! Januar könnte für einige schwierig werden, da dann auch immer die NAMM ist. Ich würde eher den Februar vorschlagen.
